connect/src/main/java/com/stripe/android/connect/webview/StripeConnectWebViewContainerController.kt

```diff
@@ -125,6 +132,39 @@ internal class StripeConnectWebViewContainerController<Listener : StripeEmbedded
         return embeddedComponentManager.getInitialParams(context)
     }
 
+    /**
+     *
+     */
+    suspend fun onPermissionRequest(context: Context, request: PermissionRequest) {
+        // we only care about camera permissions (audio/video)
+        val permissionsRequested = request.resources.filter {
+            it in listOf(PermissionRequest.RESOURCE_AUDIO_CAPTURE, PermissionRequest.RESOURCE_VIDEO_CAPTURE)
+        }.toTypedArray()
+        if (permissionsRequested.isEmpty()) {
+            request.deny() // no supported permissions were requested, so reject the request
+            logger.debug(
+                "($loggerTag) Denying permission - ${request.resources.joinToString()} are not supported"
+            )
```

Comment:
Good call. We'll want an analytic event too now that I think about it so that we know if an embedded component is expecting any unsupported permissions. I've added a TODO for when we add analytics (hopefully tomorrow or Friday)